src/NuGet.Licenses/license-text-extractor.ps1

```diff
@@ -15,7 +15,7 @@ Write-Host "Found $($licenseIds.Count) licenses."
 # The SPDX version that client uses is here: https://github.com/NuGet/NuGet.Client/blob/release-6.4.x/src/NuGet.Core/NuGet.Packaging/Licenses/NuGetLicenseData.cs
 # Make note of the release label in the NuGet.Client URL above! This should match the version of NuGet.Packaging that NuGet.Licenses.csproj depends on.
 Write-Host "Fetching exception IDs..."
-$exceptionResponseData = Invoke-RestMethod "https://raw.githubusercontent.com/spdx/license-list-data/45d10da0366f5fa931f60f3931fd23d5fb708de5/json/exceptions.json"
```

Comment:
I think we should use specific latest commit hash instead of  just main.

Reply:
^ @NuGet/gallery-team
What do you think?

Reply:
Note that this change was made in line with https://github.com/NuGet/NuGet.Licenses/blob/main/src/NuGet.Licenses/license-text-extractor.ps1#L10, which also downloads the licenses list from main.

Reply:
I'll wait for @joelverhagen and @agr's feedback on this since they created the extractor tool.

Reply:
The comment starting at line 14 explains why specific version has to be used for exceptions. We can't just grab latest from `main`, we have to align with the snapshot used by the version of the `NuGet.Packaging` package referenced by the project.